### PR TITLE
model_tools: update installation section

### PR DESCRIPTION
**Pull request by jkamelin**

### CODE REVIEW

File: tools/model_tools/README.md [hunk 1]
```diff
@@ -34,10 +34,18 @@ Installation from source is as follows:
 
 1. Install Python (version 3.6 or higher), [setuptools](https://pypi.org/project/setuptools/):
 
+2. Install [openvino-dev](https://pypi.org/project/openvino-dev/) python package of the corresponding version:
+
+```sh
+pip install openvino-dev
```
Comment (vladimir-dudnik) [inline tools/model_tools/README.md:40]:
```suggestion
pip install openvino-dev[caffe,caffe2.onnx,tensorflow2,pytorch,mxnet]
```

Comment (jkamelin) [reply tools/model_tools/README.md]: Done

**Review by vladimir-dudnik — APPROVED** [review]

**Comment by vladimir-dudnik** [comment]
cc @IRDonch

File: tools/model_tools/README.md [hunk 2]
```diff
@@ -34,20 +34,24 @@ Installation from source is as follows:
...
+```sh
+pip install openvino-dev[caffe,caffe2,onnx,tensorflow2,pytorch,mxnet]
+```
+> **NOTE**: For example, if you are using OMZ Tools for 2021.4.2 then install openvino-dev==2021.4.2.
+
 2. Install the tools with the following command:
 
 ```sh
-python setup.py install
+export OMZ_ROOT=<omz_dir>
```
Comment (IRDonch) [inline tools/model_tools/README.md:47]: FYI, you should no longer need to do this when using pip 21.3+.

**Comment by Wovchena** [comment]
OMZ branching strategy has changed: https://github.com/openvinotoolkit/open_model_zoo/pull/2905. Retarget to master.

File: tools/model_tools/README.md (hunk 2 above)
Comment (vladimir-dudnik) [reply tools/model_tools/README.md:47]: @jkamelin have you address this coment?

Comment (jkamelin) [reply tools/model_tools/README.md]: Added note about pip version

**Pull request merged**